templates/index.html

```diff
@@ -32,125 +32,127 @@
+				{% if harmony_domain %}
+					<div class="col">
+						<div class="float-right top-row">
+							<span class="border"> Harmony Controls </span>
+							<button class="btn btn-primary" id="watch-a-movie">Watch a movie</button>
+							<button class="btn btn-secondary" id="off">Off</button>
 						</div>
 					</div>
+				{% endif %}
 				</div>
-  {% endfor %}
-			</div>
-			<div class="row">
-				<div class="col"align="center">
-					<span style="display: block !important; width: 320px; text-align: center; font-family: sans-serif; font-size: 12px;">
-						<img src="http://weathersticker.wunderground.com/weathersticker/cgi-bin/banner/ban/wxBanner?bannertype=wu_clean2day_metric_cond&airportcode=EGKA&ForcedCity={{city}}&ForcedState={{country}}&wmo=03874&language=EN" alt="Find more about Weather in Southampton, UK" width="300" />
-					</span>
+				<div class="row">
+					<h4 class="border">Named devices</h4>
 				</div>
-			</div>
-			<div class="row">
-				<h4>Other devices</h4>
-			</div>
-			<div class="row">
-				<div class="col">
-					<table>
-						<tr>
-							<th>Name</th>
-							<th>MAC</th>
-							<th>Last seen</th>
-							<th>Date</th>
-						</tr>
-  {% for dev in devices %}
-						<tr class="{{dev['color']}}">
-							<td>{{dev['description']}}</td>
-							<td>{{dev['mac']}}</td>
-							<td>{{dev['pretty_lastseen']}}</td>
-							<td>{{dev['lastseen']}}</td>
-						</tr>
-  {% endfor %}
-					</table>
+				<div class="row">
+	  {% for dev in named_devices %}
+					<div class="col-auto col-card">
+						<div class="card {{dev['color']}} background" style="width: 12rem;">
+							<div class="card-body">
+								<h5 class="card-title">{{dev['name']}}</h5>
+								<h6 class="card-subtitle mb-2 text-muted">{{dev['pretty_lastseen']}}</h6>
+								<p class="card-text">{{dev['lastseen']}}</p>
+							</div>
+						</div>
+					</div>
+	  {% endfor %}
 				</div>
-			</div>
-			<div class="row">
-				<div class="col">
-					<form action="/name_device" method="post">
-						<h4> Name a device </h4>
-						<p>Mac <input type="text" name="mac" />
-						</p>
-						<p>Name <input type="text" name="name" />
+				<div class="row">
+					<div class="col" align="center" id="btnOther">
+						<p>
+							<button class="btn btn-primary" type="button" data-toggle="collapse" data-target=".tools-collapse" aria-expanded="false" aria-controls="devicesPanel addPanel removePanel">
+								Other devices
+							</button>
 						</p>
-						<input type="submit" value="Submit" />
-					</form>
+					</div>
+					<div class="col" align="center">
+						<span style="display: block !important; width: 320px; text-align: center; font-family: sans-serif; font-size: 12px;">
+							<img src="http://weathersticker.wunderground.com/weathersticker/cgi-bin/banner/ban/wxBanner?bannertype=wu_clean2day_metric_cond&airportcode=EGKA&ForcedCity={{city}}&ForcedState={{country}}&wmo=03874&language=EN" alt="Find more about Weather in Southampton, UK" width="300" />
```

Comment:
Catching this just because I didn't review it last time: It says `"Find more about Weather in Southampton, UK"` in the alt text, this should either be removed or changed to {{city}} and {{country}}

Reply:
Corrected :)